library/core/registry/src/testmod/resources/quilt.mod.json

```
@@ -12,6 +12,7 @@
     "load_type": "always",
     "entrypoints": {
       "init": [
+        "org.quiltmc.qsl.registry.test.builder.RegistryLibBuiltinBuilderTest",
```

```suggestion
        "org.quiltmc.qsl.registry.test.builder.RegistryLibBuilderTests",
```
see <https://github.com/QuiltMC/quilt-standard-libraries/actions/runs/17590969310/job/49971374490#step:5:11466>

---

this is what I get for copy-pasting without checking.